src/Illuminate/Auth/AuthManager.php

```diff
@@ -329,6 +329,21 @@ public function setApplication($app)
         return $this;
     }
 
+    /**
+     * Get the guard that authenticates the user's use.
+     * 
+     * @return string|null
+     */
+    public function getGuardName()
+    {
+        foreach (array_keys($this->app['config']['auth.guards']) as $guard) {
```

Comment:
What's about custom auth guards added using `Auth::extend()`?

Reply:
If you are using multiple guards sometimes we need to find out which user using which guard. This method helps you to find out the guard name that you are using